src/sasquatchbackpack/sasquatch.py

```diff
@@ -0,0 +1,134 @@
+"""Handles dispatch of backpack data to kafka."""
+
+__all__ = ["BackpackDispatcher", "DispatcherConfig"]
+
+import os
+from dataclasses import dataclass
+from string import Template
+
+import requests
+
+from sasquatchbackpack import sources
+
+# Code yoinked from https://github.com/lsst-sqre/
+# sasquatch/blob/main/examples/RestProxyAPIExample.ipynb
+
+
+@dataclass
+class DispatcherConfig:
+    """Class containing relevant configuration information for the
+    BackpackDispatcher.
+
+    Values
+    ------
+    sasquatch_rest_proxy_url
+        environment variable contatining the target for data
+    partitions_count
+        number of partitions to create
+    replication_factor
+        number of replicas to create
+    namespace
+        environment varible containing the target namespace
+    """
+
+    sasquatch_rest_proxy_url = os.getenv(
+        "SASQUATCH_REST_PROXY_URL",
+        "https://data-int.lsst.cloud/sasquatch-rest-proxy",
+    )
+    partitions_count = 1
+    replication_factor = 3
+    namespace = os.getenv("BACKPACK_NAMESPACE", "lsst.backpack")
+
+
+class BackpackDispatcher:
+    """A class to send backpack data to kafka.
+
+    Parameters
+    ----------
+    source
+        DataSource containing schema and record data to be
+        posted to remote
+    config
+        DispatcherConfig to transmit other relevant information to
+        the Dispatcher
+    """
```

Comment:
For the parameters, we follow the Numpydoc style guide, you usually include the parameter name and data type.
See for example the [developer guide ](https://developer.lsst.io/python/numpydoc.html#documenting-python-apis-with-docstrings)

So here it would be:

```
    Parameters
    -----------
    source : Datasource
        DataSource containing schema and record data to be
        posted to remote
    config : DispatcherConfig
        Other relevant configuration to the Dispatcher
```